src/services/userService.js

```
@@ -0,0 +1,33 @@
+'use strict';
+
+import { User } from '../models/userModel.js';
+
+const getAll = async () => {
+  const result = await User.findAll();
+
+  return result;
+};
+
+const getById = async (id) => {
+  return User.findByPk(id);
+};
+
+const create = async ({ name }) => {
+  return User.create({ name });
+};
+
+const update = async ({ name }) => {
+  return User.update({ name }, { where: { id } });
```

This `update` function is missing the `id` of the user to update. The `where: { id }` clause will fail because `id` is not defined in the function's scope. You need to pass the `id` into this function, similar to how it's done in the `remove` function. The function signature should likely be `const update = async ({ id, name }) => ...`.